client/landing/stepper/index.tsx

```diff
@@ -58,16 +59,11 @@ function initializeCalypsoUserStore( reduxStore: any, user: CurrentUser ) {
 	reduxStore.dispatch( setCurrentUser( user ) );
 }
 
-function determineFlow() {
-	const flowNameFromPathName = window.location.pathname.split( '/' )[ 2 ];
```

Comment:
No need to keep this flow, since a similar (but more refined) logic is already being used when invoking `getFlowFromURL`